schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonOpenDataSoft.xsl

```diff
@@ -264,15 +268,84 @@
+                        topic="environment"/>
+              <odstheme label="Transports, Déplacements"
+                        topic="transportation"/>
+              <odstheme label="Aménagement du territoire, Urbanisme, Bâtiments, Equipements, Habitat"
+                        topic="structure"/>
+              <odstheme label="Economie, Entreprise, PME, Développement économique, Emploi"
+                        topic="economy"/>
+              <odstheme label="Administration, Gouvernement, Finances publiques, Citoyenneté"
+                        topic="society"/>
+              <odstheme label="Justice, Sécurité, Police, Criminalité"
+                        topic="society"/>
+              <odstheme label="Sports, Loisirs"
+                        topic="society"/>
+              <odstheme label="Hébergement, industrie hôtelière"
+                        topic="society"/>
+              <odstheme label="Services sociaux"
+                        topic="society"/>
+
+              <odstheme label="Health"
+                        topic="health"/>
+              <odstheme label="Culture, Heritage"
+                        topic="society"/>
+              <odstheme label="Education, Training, Research, Teaching"
+                        topic="society"/>
+              <odstheme label="Environment"
+                        topic="environment"/>
+              <odstheme label="Transport, Movements"
+                        topic="transportation"/>
+              <odstheme label="Spatial planning, Town planning, Buildings, Equipment, Housing"
+                        topic="structure"/>
+              <odstheme label="Economy, Business, SME, Economic development, Employment"
+                        topic="economy"/>
+              <odstheme label="Administration, Government, Public finances, Citizenship"
+                        topic="society"/>
+              <odstheme label="Justice, Safety, Police, Crime"
+                        topic="society"/>
+              <odstheme label="Sports, Leisure"
+                        topic="society"/>
+              <odstheme label="Accommodation, Hospitality Industry"
+                        topic="society"/>
+              <odstheme label="Services, Social"
+                        topic="society"/>
+            </xsl:variable>
+
+            <xsl:variable name="odsThemes"
+                          select="metas/theme|dataset/metas/default/theme"/>
+            <xsl:if test="count($odsThemes) > 0">
+              <xsl:for-each select="distinct-values($odsThemeToIsoTopic[@label = $odsThemes]/@topic)">
+                <mri:topicCategory>
+                  <mri:MD_TopicCategoryCode>
+                    <xsl:value-of select="."/>
+                  </mri:MD_TopicCategoryCode>
+                </mri:topicCategory>
+              </xsl:for-each>
+
+              <mri:descriptiveKeywords>
+                <mri:MD_Keywords>
+                  <xsl:for-each select="$odsThemes">
```

Comment:
I thought we were only putting the free text values here? looks like the recognized themes will be duplicated as it is

Reply:
Santé is matching health topic category. So we end up with a proper topic and we keep the ODS theme in  keywords if you still want to search by ODS themes.

```xml
<mri:topicCategory>
<mri:MD_TopicCategoryCode>health</mri:MD_TopicCategoryCode>
</mri:topicCategory>
<mri:descriptiveKeywords>
<mri:MD_Keywords>
<mri:keyword>
<gco:CharacterString>Santé</gco:CharacterString>
</mri:keyword>
<mri:type>
<mri:MD_KeywordTypeCode codeListValue="theme" codeList="./resources/codeList.xml#MD_KeywordTypeCode"/>
</mri:type>
</mri:MD_Keywords>
</mri:descriptiveKeywords>
```

Looks fine?